highlight.io/components/Home/FeatureCarousel/FeatureCarousel.tsx

```diff
@@ -113,14 +116,34 @@ const features: Feature[] = [
 		featureImage3: <HiDatabase className="h-[20px] w-[20px]" />,
 		link: '/logging',
 	},
+	{
+		name: 'Traces',
+		title: 'Traces',
+		description:
+			'Get performance insights on requests and transactions throughout your web application stack.',
+		thumbnail: <HiSparkles className="h-[35px] w-[35px]" />,
+		desktopImage: traces,
+		mobileImage: tracesscreenshot,
+		right: true,
+		shortenWidth: true,
+		beta: true,
+		feature1: 'Powerful Visualization Capabilities',
+		featureImage1: <HiTemplate className="h-[20px] w-[20px]" />,
+		feature2: 'Distributed Tracing Support',
+		featureImage2: <HiGlobe className="h-[20px] w-[20px]" />,
+		feature3: 'OpenTelemetry Support',
+		featureImage3: <HiExclamationCircle className="h-[20px] w-[20px]" />,
+		link: '/docs/getting-started/tracing',
```

Comment:
Should this link be active? Think I am getting a 404 when clicking the "Learn more" link

Reply:
Link should be active now